jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -87,11 +87,12 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                         "TEST_TYPE=$test_type",
                         "TEST_STACK=$test_stack",
                         "EXTRA_HW=$extra_hw",
+                        "LABEL=$label_name",
                         "KEEP_FAILURE_ENV=${env.KEEP_FAILURE_ENV}",
                         "KEEP_MINUTES=${env.KEEP_MINUTES}"]
                     ){
                         try{
-                            timeout(60){
+                            timeout(90){
```

Comment:
I should change that back. These optimizations bring the execution time down to 45 minutes.